pincer/objects/guild/guild.py

```diff
@@ -474,19 +478,340 @@ async def modify_member(
             Whether the member is deafened |default| :data:`None`
         channel_id : Optional[:class:`~pincer.utils.snowflake.Snowflake]
             Voice channel id to move to |default| :data:`None`
+        reason : Optional[:class:`str`]
+            audit log reason |default| :data:`None`
+            
         Returns
         -------
         :class:`~pincer.objects.guild.member.GuildMember`
             The new member object.
         """
         ...
 
-    async def modify_member(self, _id: int, **kwargs) -> GuildMember:
+    async def modify_member(self, _id: int, reason=None, **kwargs) -> GuildMember:
```

Comment:
missing `reason` type

Reply:
since it is overloaded, types are not needed at all